src/helm/benchmark/model_metadata_registry.py

```diff
@@ -0,0 +1,70 @@
+"""This allows users to register additional models via configuration file.
+
+TODO(#1673): Support adding other kinds of models besides AI21.
+"""
+
+from typing import Any, Dict, Iterable, Optional, List
+from dataclasses import dataclass, field
+from datetime import date
+
+import cattrs
+import dacite
+import yaml
+from helm.benchmark.presentation.schema import ModelField
+
+from helm.common.general import parse_hocon
+from helm.proxy.models import ALL_MODELS, FULL_FUNCTIONALITY_TEXT_MODEL_TAG, MODEL_NAME_TO_MODEL, TEXT_MODEL_TAG, Model
+
+
+@dataclass(frozen=True)
+class ModelMetadata:
+    name: str
+
+    # Organization that originally created the model (e.g. "EleutherAI")
```

Comment:
Do we want these comments to be `"""` under the field so that they get pulled into documentation?

Reply:
Done.